### DISCUSSION TITLE
Scrape errors on Bunkrr

### DISCUSSION THREAD

**Discussion by katoonnoass** [jbsparrow/CyberDropDownloader#247] [Q&A] [answered]
Recently I formatted my computer and I changed systems from Windows 10 to Windows 11, after installing Python and cryberdrop I can't download from bunkr, it always reports a screpe error.

Logs

[downloader.log](https://github.com/user-attachments/files/17670809/downloader.log)
[Download_Error_URLs.csv](https://github.com/user-attachments/files/17670810/Download_Error_URLs.csv)
[Last_Scraped_Forum_Posts.txt](https://github.com/user-attachments/files/17670811/Last_Scraped_Forum_Posts.txt)
[Scrape_Error_URLs.csv](https://github.com/user-attachments/files/17670812/Scrape_Error_URLs.csv)
[Unsupported_URLs.txt](https://github.com/user-attachments/files/17670813/Unsupported_URLs.txt)

**Comment by NTFSvolume** [comment, marked as answer]
You have to update. You are using a very old version, `5.3.31`

Latest version is `5.7.1`. Check the wiki and releases page

**Comment by katoonnoass** [comment]
Strange, I had formatted the computer, I had downloaded the wiki and this version came, there must be a bug, I reinstalled it again and it went to the new version by itself, thank you!
